### DISCUSSION TITLE
How to retrieve rows from jsonb type containing array of strings?

### DISCUSSION THREAD

**Discussion by caesar003** [supabase/supabase#11638] [Questions]
Hi, everyone.
I'm kinda new to supabase, and wondering if anyone could possibly help me out.

Let's say I have table of movies that looks like this,
```
{
  title: "12 Angry Men",
  year: 1957,
  genre: ["Crime", "Drama"],
  director: "Sidney Lumet",
  casts: ["Henry Fonda", "Lee J. Cobb", "Joseph Sweeney", ],
}
```
with those `genre` and `casts` columns with data type of jsonb,
My question is, what should I do if I need to retrieve rows with specific genre, or movies with specific actor, while my approach with `...from("movies").select().ilike("genre", "%comedy%")` does give me this kind of error

```
{
    "error": {
        "code": "42883",
        "details": null,
        "hint": "No operator matches the given name and argument types. You might need to add explicit type casts.",
        "message": "operator does not exist: jsonb ~~* unknown"
    },
    "data": null,
    "count": null,
    "status": 404,
    "statusText": "Not Found"
}
```

Thank you

**Comment by steve-chavez** [comment]
Check the contains filter https://supabase.com/docs/reference/javascript/contains
